luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/SingleDataSourcePhysicalTableFactory.java

```diff
@@ -110,6 +113,15 @@ LuthierPhysicalTableParams buildParams(
                 )
         );
         params.metadataService = resourceFactories.getMetadataService();
+        // registers each table we build in the metadataService
+        params.metadataService.update(
```

Comment:
It seems to me that we should be doing this in the LuthierIndustrialPark, rather than
buried in this params class. For one thing, I'm not comfortable with embedding side effects in
the construction of what appears to be a dumb bean. For another, since at the end of the day
the LuthierIndustrialPark is responsible for making sure everything is built, I think it
makes sense that it should have the responsibility to make sure that everything that should
know about our physical tables, knows about them.

Reply:
done.